IMPORTANT

```diff
@@ -43,12 +47,11 @@ name changing command. :)
 The *MOST* important thing to remember though is; RTFM!!  Read the
 documents supplied with the distribution, look at the code, try something
 out.. Only then if that fails should you go asking questions on boards and
-such.  It also helps if you give an example of what you've tried too.
+such.  It also helps if you give an example of what you've tried, too.
 Bug reports I'm still up to receiving, but even then, don't be upset if
 you don't get a thorough response.
 
 
-                                                              Andrew Collington
-                                                    The Amnuts Development Team
```

Comment:
oh no, the development team has been removed and we jumped ahead a few decades

Reply:
I can't do anything about the march of time - it really has been a couple decades since that was last updated!  But it wasn't cool for me to remove the dev team line, so that'll go back in once I'm in front of my computer again!

Reply:
Fixed in https://github.com/amnuts/Amnuts/pull/12/commits/a7ce2cf4d9560297cff7e2c297f1e50582793746